extruct/service.py

```diff
@@ -20,9 +20,10 @@ def _decorated(*args, **kwargs):
     return _decorated
 
 
-def async_extruct(url, microdata=True, jsonld=True, rdfa=True):
+def async_extruct(url, microdata=True, jsonld=True, rdfa=True,
+                  opengraph=True, microformat=True):
     response.content_type = 'application/json'
-    result = metadata_from_url(url, microdata, jsonld, rdfa)
+    result = metadata_from_url(url, microdata, jsonld, rdfa, opengraph, microformat)
```

Comment:
Would be good to use keyword arguments here as well, as it's easy to confuse the order.